v2.1/admin-ui-statements-page.md

```diff
@@ -0,0 +1,110 @@
+---
+title: Statements Page
+toc: false
+---
+
+The **Statements** page helps you identify the frequently executed or high latency [SQL statements](sql-statements.html). The **Statements** page also allows you to drill down to the details of an individual SQL statement by clicking on the statement to view the **Statement Details** page.
+
+To view the **Statements** page, open [http://localhost:8080/#/statements](http://localhost:8080/#/statements) in your browser (replacing `localhost` and `8080` with your node's host and port).
+
+<div id="toc"></div>
+
+{{site.data.alerts.callout_danger}}
+**This feature is a work in progress**. It will change leading up to the v2.1 release.
+{{site.data.alerts.end}}
+
+## Limitations
+
+- If you have multiple applications running on the cluster, the **Statements** page shows cumulative parameter values across all applications, while the **Statements Details** page shows the values for the first application only.
+- The **Statements** page provides the SQL statement details only for the [gateway node](architecture/sql-layer.html#overview). To view the details for other nodes, [access the Admin UI](admin-ui-access-and-navigate.html#access-the-admin-ui) from that node and navigate to `http://<node address>:8080/#/statements` from the browser.
+- The **Statements** page displays the details of the SQL statements executed only within a specified time interval. By default, the time interval is set to one hour; however, you can customize the interval using the [`diagnostics.reporting.interval`](cluster-settings.html#settings) cluster setting.
+
+## Understanding the Statements page
+
+### SQL statement fingerprint
+
+Whenever possible, the **Statements** page displays the details of SQL statement fingerprints instead of individual SQL statements.
```

Comment:
I feel like this is a little bit confusing.  The way I think about it is that there are some statements where the fingerprint happens to be the same as the statement itself, notably when they contain no constant values.  We're still using the fingerprint.  But this sentence makes it sound less orderly or something...

Reply:
@couchand Hmmm....right. So effectively, even if the fingerprint is the statement itself, it is still a fingerprint. I think I can remove the "Whenever possible" part. Right?